stoplight.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function(){
+
+
+$("#stoplight" > "#red").on("Click", function(){
+   $(".bulb active").css("background-color", "yellow");
+
```

Comment:
the first step is just getting the click event to fire and seeing if that works. small incremental steps.